src/core/Config.cpp

```diff
@@ -130,109 +411,65 @@ Config::Config(const QString& fileName, QObject* parent)
 Config::Config(QObject* parent)
     : QObject(parent)
 {
-    // Check if portable config is present. If not, find it in user's directory
-    QString portablePath = QCoreApplication::applicationDirPath() + "/keepassxc.ini";
+    // Check if portable config is present (use it also to store local config)
+    QString portablePath = QDir::fromNativeSeparators(QCoreApplication::applicationDirPath()) + "/keepassxc.ini";
     if (QFile::exists(portablePath)) {
         init(portablePath);
-    } else {
-        QString userPath;
-        QString homePath = QDir::homePath();
-
-#if defined(Q_OS_UNIX) && !defined(Q_OS_MACOS)
-        // we can't use QStandardPaths on X11 as it uses XDG_DATA_HOME instead of XDG_CONFIG_HOME
-        QByteArray env = qgetenv("XDG_CONFIG_HOME");
-        if (env.isEmpty()) {
-            userPath = homePath;
-            userPath += "/.config";
-        } else if (env[0] == '/') {
-            userPath = QFile::decodeName(env);
-        } else {
-            userPath = homePath;
-            userPath += '/';
-            userPath += QFile::decodeName(env);
-        }
+        return;
+    }
 
-        userPath += "/keepassxc/";
+    QString configPath;
+    QString localConfigPath;
+
+#if defined(Q_OS_WIN)
+    configPath = QDir::fromNativeSeparators(QStandardPaths::writableLocation(QStandardPaths::AppDataLocation));
```

Comment:
Can remove all `fromNativeSeparators` since we standardize everything before return with `toNativeSeparators`